<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="login_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>login_name</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_pwd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>login_pwd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>login_btn_txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel_btn_txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="find_pwd">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>find_pwd_label_txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="reg_account">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>to_register_label_txt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>221</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>221</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="verify_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="verify_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>change_verify_code_label</string>
    </property>
   </widget>
   <widget class="Verification" name="widget_verification" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Verification</class>
   <extends>QWidget</extends>
   <header location="global">verification.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
